### ISSUE TITLE
nDCG loss and metric disagree

### ISSUE THREAD

**Issue by eladBenedict** [tensorflow/ranking#166]
Hi,

I'm training a model with approximated nDCG as the loss, and am evaluating the test set on nDCG as well.

It seems that the loss decreases during training on both the train and the test set, however the metric shows an opposite effect. 

![image](https://user-images.githubusercontent.com/11731485/74089455-fc3ce880-4aa9-11ea-8e30-7d17f28232c6.png)

![image](https://user-images.githubusercontent.com/11731485/74089456-00690600-4aaa-11ea-8d32-7f6d5fcfde10.png)

Please advise.

Thanks,
Elad

**Comment by LiangqunLu** [comment]
What is your group_size and what is your loss function?

**Comment by xuanhuiwang** [comment]
@eladBenedict, is this still an issue. It sounds that there is some setup issue for your case.

**Comment by eladBenedict** [comment]
@LiangqunLu the group_size is 1 and the loss is approx_ndcg
